### PR TITLE
added macro {php …} as replacement for {? …}

### LINKED ISSUE #207: Latte: makro {? se změnilo na {php
viz: added macro {php …} as replacement for {? …}

### PR DESCRIPTION
**Pull request by pilec**
[Closes #207]

### CODE REVIEW

**Comment by klimesf** [comment]
:+1:

File: cs/default-macros.texy [hunk 1]
```diff
@@ -42,7 +42,7 @@ Zajímají vás bližší informace o [syntaxi Latte |templating#latte] nebo o [
 |---------------------------
 | `{include 'file.latte'}`       | [načte šablonu z dalšího souboru |#Vložení souboru]
 | `{cache $key} … {/cache}`      | [cachuje část šablony |caching#cachovani-v-sablonach]
-| `{? expression}`               | [vyhodnotí výraz, ale nevypíše |#Vyhodnocení kódu]
+| `{php expression}`               | [vyhodnotí výraz, ale nevypíše |#Vyhodnocení kódu]
```
Comment (Majkl578) [inline cs/default-macros.texy]: indent

**Comment by Majkl578** [comment]
`{? ...}` was not deprecated, so probably both should stay.

File: cs/default-macros.texy (hunk 1 above)
Comment (pilec) [inline cs/default-macros.texy]: what kind of?

**Comment by pilec** [comment]
imho is more obvious for beginners, dont you think?

File: cs/default-macros.texy (hunk 1 above)
Comment (JanTvrdik) [inline cs/default-macros.texy]: The `|` character is not aligned.

**Comment by JanTvrdik** [comment]
`{php ...}` should be in the table, `{? ...}` should be mentioned in text probably in `.[note]`.

File: cs/default-macros.texy (hunk 1 above)
Comment (pilec) [inline cs/default-macros.texy]: :+1:

**Comment by pilec** [comment]
note added

**Comment by Majkl578** [comment]
@klimesf: Next time please don't merge PRs with multiple "typo" commits, but squash them locally and cherry-pick the resulting one.

**Comment by klimesf** [comment]
@Majkl578: Sorry, I didn't notice.

**Pull request merged**